lib/katakata_irb/completor.rb

```diff
@@ -271,9 +271,15 @@ def self.analyze(code, binding = empty_binding)
       [:ivar, name, calculate_scope.call.self_type] if icvar_available
     in [:var_ref, [:@cvar,]]
       [:cvar, name, calculate_scope.call.self_type] if icvar_available
-    in [:call, receiver, [:@period,] | [:@op, '&.',] | :'::' => dot, [:@ident | :@const,]]
+    in [:call, receiver, [:@op, '::',] | :'::', [:@ident | :@const,]]
```

Comment:
This PR contains https://github.com/tompng/katakata_irb/pull/22 to pass CI with Ruby3.3.